Error while building sched

```
➜  sched git:(master) ✗ godep go build .
../common/store/etcd/db.go:9:2: cannot find package "github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context" in any of:
    /usr/local/go/src/github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context (from $GOROOT)
    /Users/pradeep/src/github.com/mesos/mr-redis/sched/Godeps/_workspace/src/github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context (from $GOPATH)
    /Users/pradeep/src/github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context
godep: go exit status 1
```

Looks like wrong import.

--- a/common/store/etcd/db.go
+++ b/common/store/etcd/db.go
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context"
+	"golang.org/x/net/context"
 	cli "github.com/coreos/etcd/client"
 )
 

